Emby.Server.Implementations/Session/SessionManager.cs

```diff
@@ -1363,9 +1363,26 @@ public Task SendServerShutdownNotification(CancellationToken cancellationToken)
         {
             CheckDisposed();
 
+            CloseAllWebSockets(cancellationToken).ConfigureAwait(false);
```

Comment:
```suggestion
            await CloseAllWebSockets(cancellationToken).ConfigureAwait(false);
```

will also need to add `async` to the function signature